.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember-suave/recommended'
```

Comment:
I'm not a huge fan of the forced destructuring of `Ember` since it currently has no advantages over not doing it. It is not clear yet whether that will actually allow tree shaking to be easier in the future, but it makes the code a little harder to read sometimes.